src/SFA.DAS.Provider.PR.Web/Validators/RequestPermissionsSubmitModelValidator.cs

```diff
@@ -0,0 +1,34 @@
+﻿using FluentValidation;
+using SFA.DAS.Provider.PR.Web.Models;
+
+namespace SFA.DAS.Provider.PR.Web.Validators;
+
+public class RequestPermissionsSubmitModelValidator : AbstractValidator<RequestPermissionsSubmitModel>
+{
+    public const string RequestPermissionsAddRecordsErrorMessage = "Select the permissions you want to set for Add apprentice records";
+    public const string RequestPermissionsRecruitApprenticesErrorMessage = "Select the permissions you want to set for Recruit apprentices";
+    public const string MatchesExistingPermissionErrorMessage = "These are your current permissions. Select new permissions";
+
```

Comment:
I might be mistaken, but I believe on this page you can never have empty permission values so I think the first two tests are not required (and the associated messages)